components/MarkdownTextBlock/samples/MarkdownTextBlockCustomSample.xaml.cs

```diff
@@ -0,0 +1,613 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.Labs.WinUI.MarkdownTextBlock;
+
+namespace MarkdownTextBlockExperiment.Samples;
+
+/// <summary>
+/// An example sample page of a custom control inheriting from Panel.
+/// </summary>
+[ToolkitSample(id: nameof(MarkdownTextBlockCustomSample), "Custom control", description: $"A sample for showing how to create and use a {nameof(CommunityToolkit.Labs.WinUI.MarkdownTextBlock)} custom control.")]
+public sealed partial class MarkdownTextBlockCustomSample : Page, INotifyPropertyChanged
+{
+    private MarkdownConfig _config;
+    private const string _markdown = @"
```

Comment:
You should be able to use the new raw string literal here so that you don't have to escape everything.

Reply:
The escapes are just for escaping markdown. So they are needed anyway.